@here/harp-examples/src/datasource_satellitetile.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { GeoCoordinates } from "@here/harp-geoutils";
+import { MapControls, MapControlsUI } from "@here/harp-map-controls";
+import { CopyrightElementHandler, MapView } from "@here/harp-mapview";
+import { WebTileDataSource } from "@here/harp-webtile-datasource";
+import { appCode, appId } from "../config";
+
+// tslint:disable:max-line-length
+/**
+ * A simple example using the webtile data source. Tiles are retrieved from
+ * ```
+ * https://1.aerial.maps.api.here.com/maptile/2.1/maptile/newest/satellite.day/${level}/${column}/${row}/512/png8?app_id=${appId}&app_code=${appCode}
+ * ```
+ *
+ * A [[WebTileDataSource]] is created with specified applications' appId and appCode passed
+ * as [[WebTileDataSourceOptions]]
+ * ```typescript
+ * [[include:harp_gl_datasource_satellitetile_1.ts]]
+ * ```
+ * Then added to the [[MapView]]
+ * ```typescript
+ * [[include:harp_gl_datasource_satellitetile_2.ts]]
+ * ```
+ */
+export namespace WebTileDataSourceExample {
+    // creates a new MapView for the HTMLCanvasElement of the given id
+    export function initializeMapView(id: string): MapView {
+        const canvas = document.getElementById(id) as HTMLCanvasElement;
+
+        const map = new MapView({
+            canvas,
+            theme: "resources/berlin_tilezen_base.json"
+        });
+        map.setCacheSize(100, 100);
```

Comment:
why is this needed?

Reply:
Was already in the previous one, so I did not change it